src/plugins/data/server/search/strategies/ese_search/request_utils.ts

```diff
@@ -50,6 +51,8 @@ export async function getDefaultAsyncSubmitParams(
   return {
     // TODO: adjust for partial results
     batched_reduce_size: searchConfig.asyncSearch.batchedReduceSize,
+    // Decreases delays due to network when using CCS
+    ccs_minimize_roundtrips: false,
```

Comment:
QQ: if I understand correctly it should be set to true to decrease delays?

Reply:
That's correct. I believe Kibana wants to set it to `true` from now on: https://github.com/elastic/kibana/issues/159706

The change on the ES side (https://github.com/elastic/elasticsearch/pull/96012) now supports minimizing round trips with async_search, but still leaves `ccs_minimize_rountrips=false` as the default.